src/components/PoiPopup.jsx

```diff
@@ -1,8 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PoiItem from './PoiItem';
 import { fire } from 'src/libs/customEvents';
+import ActionButtons from '../panel/poi/ActionButtons';
+import nconf from '@qwant/nconf-getter';
+import Telemetry from '../libs/telemetry';
+import { addToFavorites, isInFavorites, removeFromFavorites } from '../adapters/store';
 
 const PoiPopup = ({ poi }) => {
+  const [inFavorites, setInFavorites] = useState(isInFavorites(poi));
+  const isDirectionActive = nconf.get().direction.enabled;
```

Comment:
There is now a `useConfig` hook as a simpler and more react-ish alternative to nconf. You can already find it used elsewhere in the project.

Reply:
thanks! should I create a ticket to remove all the remaining nconf occurrences?

Reply:
I'd say we don't need one. There is only one occurrence left in React components, in PoiPanel, and it can't be changed until it has been turned to a functional component… which I'm working on :slightly_smiling_face: .

Reply:
er, I meant these: 
![image](https://user-images.githubusercontent.com/1225909/123764381-06d96700-d8c5-11eb-8af3-958a75e5d8f8.png)

Reply:
They don't need to be modified nor become React components, helpers like util functions and data processing don't need to be React :) And we can't remove `nconf` completely. `useConfig` uses it behind the scene, it's just syntaxic sugar to avoid  doing `nconf.get().foo…` everywhere.